style.css

```diff
@@ -106,8 +110,118 @@
     .btn{
```

Comment:
The divider between two rows is not matching up with design. 
![Screenshot 2023-03-13 at 2 48 21 PM](https://user-images.githubusercontent.com/38748298/224658431-282d8909-f93a-4d22-a969-504fc68271a9.png)